cli/mender-artifact/main.go

```diff
@@ -395,8 +395,8 @@ func getCliContext() *cli.App {
 
 	install := cli.Command{
 		Name:        "install",
-		Usage:       "install -m <permissions> <hostfile> [artifact|sdimg|uefiimg]:<filepath>",
-		Description: "Installs a file from the host filesystem to the artifact or sdimg.",
+		Usage:       "install -m <permissions> <hostfile> [artifact|sdimg|uefiimg]:<filepath> or install -d [artifact|sdimg|uefiimg]:<directory>",
+		Description: "Installs a file or directory from the host filesystem to the artifact or sdimg.",
```

Comment:
Suggest rephrasing to `Installs a directory, or a file from the host filesystem, to the artifact or sdimg`.

Reply:
Fixed the phrasing